OrbitClientGgp/main.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+//#include <iostream>
+//#include <string>
+
+#include "absl/flags/flag.h"
+#include "absl/flags/parse.h"
+//#include "absl/flags/usage.h"
+//#include "absl/flags/usage_config.h"
+//#include "absl/strings/str_format.h"
+
+#include "ClientGgp.h"
+//#include "OrbitBase/Logging.h"
+#include "OrbitBase/ThreadPool.h"
+
+ABSL_FLAG(uint64_t, grpc_port, 44765, "Grpc service's port");
+ABSL_FLAG(int32_t, pid, 0, "pid to capture");
+ABSL_FLAG(uint32_t, capture_length, 10, "duration of capture in seconds");
+ABSL_FLAG(uint16_t, sampling_rate, 1000,
+          "Frequency of callstack sampling in samples per second");
+ABSL_FLAG(bool, frame_pointer_unwinding, false,
+          "Use frame pointers for unwinding");
+
+int main(int argc, char** argv) {
+  absl::ParseCommandLine(argc, argv);
+  uint64_t grpc_port_ = absl::GetFlag(FLAGS_grpc_port);
```

Comment:
Remove underscore at the end - that's for member variables only.

Reply:
True, I didn't realise, thanks!